src/wallet/rpcwallet.cpp

```diff
@@ -430,9 +437,15 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
             "       \"" + FeeModes("\"\n\"") + "\""},
                     {"avoid_reuse", RPCArg::Type::BOOL, /* default */ "true", "(only available if avoid_reuse wallet flag is set) Avoid spending from dirty addresses; addresses are considered\n"
             "                             dirty if they have previously been used in a transaction."},
+                    {"verbose", RPCArg::Type::BOOL, /* default */ "false",
+                            "Optional, If true, return extra information about the transaction."},
                 },
                 RPCResult{
-                    RPCResult::Type::STR_HEX, "txid", "The transaction id."
+                    RPCResult::Type::OBJ, "", "",
```

Comment:
yay for object instead of string result, can add more things going forward.

Reply:
This will need to be two RPCResults (one for verbose=False and one for verbose=True). Refer to other places in the code base on how to do that.